htdocs/install.sh

```diff
@@ -392,14 +394,22 @@ echo
 echo '*** Enabling and starting ZeroTier service...'
 
 if [ -e /usr/bin/systemctl -o -e /usr/sbin/systemctl -o -e /sbin/systemctl -o -e /bin/systemctl ]; then
-	$SUDO systemctl enable zerotier-one
-	$SUDO systemctl start zerotier-one
-	if [ "$?" != "0" ]; then
+	if [[ -d /run/systemd/system ]]; then
+		$SUDO systemctl enable zerotier-one
+		$SUDO systemctl start zerotier-one
+		if [ "$?" != "0" ]; then
+			echo
+			echo '*** Package installed but cannot start service! You may be in a Docker'
+			echo '*** container or using a non-standard init service.'
+			echo
+			exit 1
+		fi
+	else
 		echo
-		echo '*** Package installed but cannot start service! You may be in a Docker'
-		echo '*** container or using a non-standard init service.'
+		echo '*** Package installed but cannot start service! You may be in a Docker container'
+		echo '*** or using a non-standard init service.'
 		echo
-		exit 1
+		exit 0
```

Comment:
?

Reply:
If systemd isn't running, we don't want to call systemctl commands, or return an error.